pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -49,6 +49,7 @@ func NewCSRApprovingController(client clientset.Interface, csrInformer certifica
 		recognizers: recognizers(),
 	}
 	return certificates.NewCertificateController(
+		"sar-approver",
```

Comment:
should this match the name used in the controller manager (e.g. `csrapproving`)?